### DISCUSSION TITLE
User manual

### DISCUSSION THREAD

**Discussion by simjanos-dev** [simjanos-dev/LinguaCafe#202] [Ideas]
I've opened this discussion, because it is on open-ended feature, that will always need improvement.

When v0.10 releases, the current online manual will be replaced with [GitHub Wiki](https://github.com/simjanos-dev/LinguaCafe/wiki), and it will also be displayed inside the software. People will have a much easier time editing a markdown, than Vuetify pages. I've added a small guide to the [contributing](https://github.com/simjanos-dev/LinguaCafe/blob/dev/CONTRIBUTING.md) file in the dev branch.

**Comment by simjanos-dev** [comment]
@sergiolaverde0 
I've tagged you, because you were interested in modifying the user manual, so you know about this change.

@uzluisf
I've fixed every visual issue with the markdown rendering in vue, and also fixed the flag images both in the software and on the wiki.

I've not finished reading through it, but one thing I found was I don't see a distinction between Usage and Features. I think maybe we could separate pages in them based on which are related to installing, hosting and setting up linguacafe(like Importing dictionaries, Jellyfin, Backup), and to others which are related to actually using it.

And thank you again, for making it!

**Reply by uzluisf** [reply]
>I've not finished reading through it, but one thing I found was I don't see a distinction between Usage and Features. I think maybe we could separate pages in them based on which are related to installing, hosting and setting up linguacafe(like Importing dictionaries, Jellyfin, Backup), and to others which are related to actually using it.

I think whatever you think works best is the way to go. That was simply a preliminary attempt at getting the manual into a single place so I think reorganizing pages, rewriting them if needed, etc as you see fit is good. 

>And thank you again, for making it!

Thanks to you for acting on it so quickly!

**Reply by simjanos-dev** [reply]
I've read through the whole thing, it is much more well written and styled than it was before, thank you! I've changed a few things, and added more information. I've made it live on the github wiki. I'll remove it from the overview github page tomorrow, and it will be in the software in v0.10.
